### PR TITLE
cassandra/postgres get requests work

### PR DESCRIPTION
**Pull request by ucb15erickao**

### CODE REVIEW

File: database/10M.js [hunk 1]
```diff
@@ -20,21 +21,25 @@ const generateData = (file, callback) => {
...
+      const imageId = faker.random.number({ min: 1, max: 25 });
+      const image = `https://sdc-recommendations.s3-us-west-1.amazonaws.com/p${imageId}.jpg`;
       const tag = faker.random.number({ min: 0, max: 3 });
       const rating = faker.random.number({ min: 0, max: 5, precision: 0.01 });
       const reviewCount = faker.random.number({ min: 0, max: 1000 });
-      const data = `${id},${department},${category},${subcategory},${brand},${price},${title},${description},${tag},${rating},${reviewCount}\n`;
+      const data = `${id};${department};${category};${subcategory};${brand};${price};${title};${description};${image};${tag};${rating};${reviewCount}\n`;
       if (count === records) {
-        file.write(data, callback);
+        file.write(data, () => { csvFile.end(); });
```
Comment (jonfu86) [inline database/10M.js:31]: nice

File: database/dbMongo.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+const mongoose = require('mongoose');
```
Comment (jonfu86) [inline database/dbMongo.js:1]: mongo? Cool

**Review by jonfu86 — APPROVED** [review]
nice! approving and merging

**Comment by jonfu86** [comment]
Looks like you still have a conflict that must be resolved. I'll let you merge after conflict resolution

File: database/dbMongo.js (hunk 2 above)
Comment (ucb15erickao) [reply database/dbMongo.js:1]: just a copy of the original server

**Pull request merged**